src/cli/cli.js

```diff
@@ -100,19 +99,20 @@ class CLI {
           if (currentOptions.verbose) {
             if (issues.items.length > 0) {
               this.printResults(issues, processedRules, {
-                format: 'verbose'
+                format: 'verbose',
               });
             }
 
             if (processedRules.length > 0) {
               this.printResults(issues, processedRules, {
-                format: 'summary'
+                format: 'summary',
               });
             }
           } else {
-            const verboseResults = issues.items.filter(result => result.params.verbose !== undefined);
+            const verboseResults = issues.items.filter((result) => result.params.verbose !== undefined);
 
             if (verboseResults.length >= 1) {
+              // eslint-disable-next-line
```

Comment:
I would saw maybe we disable the eslint rule for console commands since this library makes a few instead of disabling it on lines?